Scripts/azure_stack_provision.ps1

```diff
@@ -56,8 +59,8 @@ if($azure_tags -ge 1) {
 }
 else {
 	Write-Host "Creating New Azure SQL server as no server found with lansa version $clone_lansa_version ..."
-    New-AzResourceGroupDeployment -ResourceGroupName dbregressiontest -TemplateFile "$git_repo_root/Template/azure/sqlserver.json" -TemplateParameterObject $azure_template_param
-    Write-Host "Created the SQL server, Now Restoring the Database $db..."
+    New-AzResourceGroupDeployment -ResourceGroupName dbregressiontest -TemplateFile "$git_repo_root/sqlserver.json" -TemplateParameterObject $azure_template_param
+    Write-Host "Created the SQL server, Now Restoring the Database from $db to $clone_lansa_version..."
     New-AzSqlDatabaseCopy -ResourceGroupName dbregressiontest -ServerName $sourceserver -DatabaseName $db -CopyResourceGroupName dbregressiontest -CopyServerName $sql_server -CopyDatabaseName $clone_lansa_version | Out-Default | Write-Host
```

Comment:
The `$sourceserver` parameter must be deleted. You must use the `$clone_lansa_Version` to locate the database to copy from.